presto-main/src/main/java/com/facebook/presto/sql/planner/QueryPlanner.java

```diff
@@ -419,6 +505,21 @@ private PlanBuilder explicitCoercionSymbols(PlanBuilder subPlan, Iterable<Symbol
                 analysis.getParameters());
     }
 
+    private ImmutableList<FilterExpressionWithAggregate> getFilterExpressions(QuerySpecification node)
+    {
+        ImmutableList.Builder resultBuilder = ImmutableList.builder();
+        analysis.getAggregates(node).stream().forEach(currentAggregate -> {
+            FilterExpressionWithAggregate filterExpressionWithAggregate = new FilterExpressionWithAggregate(
+                    currentAggregate.getFilter().isPresent() ? currentAggregate.getFilter().get() : null,
```

Comment:
Why not use `FunctionCall` instead of `FilterExpressionWithAggregate` as a parameter to your `project` method?